avm-transpiler/src/opcodes.rs

```diff
@@ -73,6 +73,7 @@ pub enum AvmOpcode {
     SHA256,   // temp - may be removed, but alot of contracts rely on it
     PEDERSEN, // temp - may be removed, but alot of contracts rely on it
     ECADD,
+    MSM,
```

Comment:
I think you'd better also add it to the hpp/cpp files now. Otherwise all toradixLEs will be wrong.